### PR TITLE
add myProfile

### PR DESCRIPTION
**Pull request by Mihir0000**

### CODE REVIEW

File: src/Components/MyProfile/MyProfile.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                })}
+              {/* {MyProfileData &&
+                show &&
+                MyProfileData.ongoing.map((item, index) => {
+                  return (
+                    <div key={index} className='col-5 m-3'>
+                      <OrderCard item={item} />
+                    </div>
+                  )
+                })} */}
```
Comment (Reeju2019) [inline src/Components/MyProfile/MyProfile.tsx:52]: Remove the comment on line 43-51.

File: src/Components/MyProfile/MyProfile.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      <Container fluid className='mt-2'>
+        <Row>
+          <Col className='col-9 firstCol'>
+            <div className='m-3 d-flex justify-content-center  '>
+              <button
+                disabled={show}
+                className={'leftButton ' + (show ? 'active' : '')}
+                onClick={showClicked}
+              >
+                Ongoing
```
Comment (Reeju2019) [inline src/Components/MyProfile/MyProfile.tsx:24]: Data should be imported from const file.

File: src/Components/MyProfile/MyProfile.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+              </button>
+              <button
+                disabled={!show}
+                className={'rightButton ' + (show ? '' : 'active')}
+                onClick={showClicked}
+              >
+                Compleated
+              </button>
```
Comment (Reeju2019) [inline src/Components/MyProfile/MyProfile.tsx:32]: Refer to the other comment

File: src/Components/MyProfile/MyProfile.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+            </div>
+          </Col>
+          <Col className='col-3 secondCol'>
+            <div className='d-flex flex-column justify-content-center m-4'>
+              <div className='profileImg align-self-center m-4'>
+                <img src='/static/my_profile/user_prof.png' className='w-100' alt='user' />
+              </div>
+              <h4 className='d-flex justify-content-center'>Name</h4>
+              <div className='d-flex flex-row justify-content-center contactNumber'>
+                +91
```
Comment (Reeju2019) [inline src/Components/MyProfile/MyProfile.tsx:62]: Refer to the other comment

File: src/Components/MyProfile/MyProfile.tsx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+                <span className='mx-1'>9876543210</span>
```
Comment (Reeju2019) [inline src/Components/MyProfile/MyProfile.tsx:63]: Refer to the other comment

File: src/Components/MyProfile/MyProfile.tsx [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+                <i className='fa-solid fa-pencil' />
+              </div>
+            </div>
+            <div className='m-2'>
+              <div className='d-flex justify-content-between'>
+                <div>Saved Address</div>
```
Comment (Reeju2019) [inline src/Components/MyProfile/MyProfile.tsx:69]: Refer to the other comment

File: src/Components/MyProfile/MyProfile.tsx [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+                <div className='update'>Update Address</div>
```
Comment (Reeju2019) [inline src/Components/MyProfile/MyProfile.tsx:70]: Refer to the other comment

File: src/Components/MyProfile/MyProfile.tsx [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+              </div>
+              <hr className='my-2' />
+              <div className='d-flex justify-content-between'>
+                <div>Contact Details</div>
```
Comment (Reeju2019) [inline src/Components/MyProfile/MyProfile.tsx:74]: Refer to the other comment

File: src/Components/MyProfile/MyProfile.tsx [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+                <div className='update'>Update</div>
```
Comment (Reeju2019) [inline src/Components/MyProfile/MyProfile.tsx:75]: Refer to the other comment

File: src/Components/MyProfile/MyProfile.tsx [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+              </div>
+              <hr className='my-2' />
+              <div>Refer a Friend</div>
```
Comment (Reeju2019) [inline src/Components/MyProfile/MyProfile.tsx:78]: Refer to the other comment

File: src/Components/MyProfile/MyProfile.tsx [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+              <hr className='my-2' />
+              <div>Get Help</div>
```
Comment (Reeju2019) [inline src/Components/MyProfile/MyProfile.tsx:80]: Refer to the other comment

File: src/Components/MyProfile/OrderCard.tsx [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+  }
+}
+
+const OrderCard: React.FunctionComponent<IOrderCardProps> = ({ item }) => {
+  return (
+    <>
+      <section className='bg-white p-3 orderCard'>
+        <div className='d-flex justify-content-between mt-3'>
+          <div className='orderId'>
+            Order ID: #<span>{item.order_id}</span>
```
Comment (Reeju2019) [inline src/Components/MyProfile/OrderCard.tsx:21]: Refer to the other comment

File: src/Components/MyProfile/OrderCard.tsx [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+            </div>
+            <div>
+              <i className='fa-solid fa-gears align-self-center mx-2' />
+              <span className='fs-5'>{item.issue}</span>
+            </div>
+          </div>
+        </div>
+        <p>{item.order_status}</p>
+        <div className='d-flex flex-row justify-content-between'>
+          <button className='callSupport'>Call Support</button>
```
Comment (Reeju2019) [inline src/Components/MyProfile/OrderCard.tsx:46]: Refer to the other comment

File: src/Components/MyProfile/OrderCard.tsx [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+          <button className='viewDetails'>View Details</button>
```
Comment (Reeju2019) [inline src/Components/MyProfile/OrderCard.tsx:47]: Refer to the other comment

**Review by Reeju2019 — CHANGES_REQUESTED** [review]

File: src/Components/MyProfile/MyProfile.tsx (hunk 1 above)
Comment (Mihir0000) [reply src/Components/MyProfile/MyProfile.tsx:52]: here I don't know what "ongoingCard" look like. this is added for "ongoingCard". also, it's needed here.

**Pull request merged**